src/commands/gin/sharing/waitReady.ts

```diff
@@ -0,0 +1,46 @@
+import { SfdxCommand } from "@salesforce/command";
+
+export default class WaitReady extends SfdxCommand {
+  public static description =
+    "Waits until all Permission Set Groups are updated";
+
+  public static examples = [
+    `$ sfdx gin:sharing:waitReady" 
```

Comment:
Nice! Can you just make the command lowercase? I remember that usually the sfdx cli enforces to be lowercase